src/features/dashboard/components/Dialogs/UpdateAppDialog/index.tsx

```diff
@@ -70,7 +70,7 @@ const UpdateAppDialog = ({ app, onClose }: IUpdateAppDialog) => {
         scopes: selectedScopes,
       });
     },
-    [updateApp],
+    [updateApp, currentLoginAccount.name],
```

Comment:
don't we need any other dependency like `data` which I see is being used inside the handler yet it's not listed in the dependency array?